src/jloptions.c

```diff
@@ -275,6 +278,7 @@ JL_DLLEXPORT void jl_parse_opts(int *argcp, char ***argvp)
         { "cpu-target",      required_argument, 0, 'C' },
         { "procs",           required_argument, 0, 'p' },
         { "threads",         required_argument, 0, 't' },
+        { "gc",              required_argument, 0, opt_gc_threads },
```

Comment:
```suggestion
        { "gc",              required_argument, 4, opt_gc_threads },
```

Let's default to 4 for CI purposes? Additionally we could add `JULIA_GC_THREADS` as an environment variable like `JULIA_NUM_THREADS` to control the number of threads being used on CI.